commands/utils/packagehandlers/gopackagehandler.go

```diff
@@ -0,0 +1,26 @@
+package packagehandlers
+
+import (
+	"github.com/jfrog/frogbot/commands/utils"
+)
+
+type GoPackageHandler struct {
+	common
+}
+
+func (golang *GoPackageHandler) UpdateDependency(fixDetails *utils.FixDetails) (bool, error) {
+	if fixDetails.DirectDependency {
```

Comment:
This is exactly why we don't need `updateDirectDependency` and `updateIndirectDependency` defined in the `PackageHandler` interface. We should keep it as a method receiver for the `GoPackageHandler` for now.